addons/redis/templates/backupactionset.yaml

```diff
@@ -119,6 +129,16 @@ spec:
       - |
         #!/bin/bash
         {{- .Files.Get "dataprotection/common-scripts.sh" | nindent 8 }}
+        {{- .Files.Get "dataprotection/switch-data-dir.sh" | nindent 8 }}
         {{- .Files.Get "dataprotection/pitr-restore.sh" | nindent 8 }}
-    postReady: []
+    postReady:
+    - job:
+        command:
+        - bash
+        - -c
+        - |
+          {{- .Files.Get "dataprotection/restore-keys.sh" | nindent 10 }}
+        image: {{ include "redis.image" . }}
```

Comment:
The CI check has failed, see https://github.com/apecloud/kubeblocks-addons/actions/runs/10937929686/job/30364901673.

This line should be `{{ include "redis7.image" . }}`, please check other places as well.